src/Maestro/tests/Maestro.ScenarioTests/TestHelpersTest.cs

```diff
@@ -27,5 +27,118 @@ public void ArgumentsWithSecretTokensInside()
 
             Assert.AreEqual("darc.exe \"-p\" \"***\" \"add-channel\" \"--github-pat\" \"***\" \"--name\" \"what-a-channel\"", formatted);
         }
+
+        [Test]
+        public void ParseOutSpecificRepoPolicy_ExactMatch()
+        {
+            string exactMatchPolicy =
```

Comment:
One of these uses a literal string with new lines, the other uses "\r\n"... Those aren't always equivalent.  We either need to normalize them, or use one or the other. :-)

In particular, when the file goes into git, it replaces the ones in the literal string with just a line feed (without the carriage return).  Which means if you sync this on a linux/mac box (or just don't tell it to do the lf => crlf on checkout), the tests will suddenly fail.

Reply:
Is there are reason we don't use the same style of string for both of them?

Reply:
Of course git does something like that. What would your recommendation be for best compatability? (Or even better, to best match what we get from darc?)

Reply:
If the format of the new lines is actually important, verbatim strings can't be used.  I'd rather we didn't compare giant strings (but that's what the tests already do).

Reply:
The format of the strings is only important in that it would be a lot of work to parse them into objects and the code for that doesn't currently exist. We should consider adding that parsing or some other kind of white space ignoring comparison for testing in the future, but it's out of scope at the moment.

Reply:
I tried running the repo policy tests locally using this branch, and I get this error: 

```
Repository policy is incorrect for all checks successful case
      Expected string length 237 but was 235. Strings differ at index 235.
      Expected: "...       "A",\r\n                     "B"\r\n                   ]\r\n", ignoring case
      But was:  "...       "A",\r\n                     "B"\r\n                   ]"
```

Reply:
That means it would also fail if run on a machine that didn't do the \n -> \r\n conversion.  It just means they need to be the same.  Either both verbatim strings (`@"string"`) or neither.

Reply:
I have them all changed over now, but I discovered a bigger problem - not all of the merge policies end in ']', which is what I was using to split them. At this point I'm writing a bunch of code to solve a test bug (since I assume that the partial string match is intended in the product code) and it's getting progressively more complicated. Unless either of you guys have a suggestion for a simpler solution I'm going to just change the branch name to be completely distinct from the one that the PowerShell test uses and add a comment to the test case explaining the situation.

Reply:
I don't think the conflict is with the powershell tests since the powershell tests use completely random branch names on every run, the conflict is between the existing tests that use what I'm going to call `base-name`, and your upcoming changes that will use `base-name`-`machine-name`. Which due to the branch matching in substring to always find the `base-name` when looking for the other. 

I think we can just delete the entry for `base-name` when the new naming is ready to be merged, and we won't hit the issue again. 

The long term fix should is probably a combination of a few things:

* Create an API + darc command to delete repository-policies
* Have the tests delete the repository-policies on cleanup.
* Implement JSON output for the get-repository-policies command so that it's easier to parse just the exact merge policy we're looking for.

Reply:
In that case I'm going to just scrap this whole thing and file a bug for the improvements. It doesn't seem like it's worth the time right now.